chord_metadata_service/experiments/schemas.py

```diff
@@ -28,15 +28,16 @@
         },
         "file_format": {
             "type": "string",
-            "enum": ["SAM", "BAM", "CRAM", "BAI", "CRAI", "VCF", "BCF", "GVCF", "BigWig", "BigBed", "FASTA",
+            "enum": ["SAM", "BAM", "CRAM", "BAI", "CRAI", "VCF", "BCF", "MAF", "GVCF", "BigWig", "BigBed", "FASTA",
                      "FASTQ", "TAB", "SRA", "SRF", "SFF", "GFF", "TABIX", "UNKNOWN", "OTHER"]
         },
         "data_output_type": {
             "type": "string",
             "enum": ["Raw data", "Derived data"]
         },
         "usage": {
-            "type": "string"
+            "type": "string",
+            "enum": ["visualized", "downloaded"]
```

Comment:
No capitalization for these values (because they are adjectives? But note that `library_layout` values are capitalized (`Single`, `Paired`)

Reply:
Capitalized them now, thanks for catching this error!